recipes/jupyterlite-javascript-kernel/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "jupyterlite-javascript-kernel" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlite_javascript_kernel-{{ version }}.tar.gz
+  sha256: 297116be6642c2f72b0b0bf3d6505efcd792b322139174d0107ae7ec4daa64f4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - hatchling >=1.6.3,<2
+    - jupyterlab >=3.5.3,<3.6.0
+    - pip
+  run:
+    - python >=3.7
+
+test:
+  commands:
+    - pip check
+  requires:
+    - pip
```

Comment:
would probably use a `grep` smoketest to ensure it gets added to `jupyter-lite.json`